### ISSUE TITLE
An error occurred when downloading video (version 2.9.4) for certain files

### ISSUE THREAD

**Issue by vijay1675** [masihyeganeh/YoutubeDownloader#52]
Hi,
I am using https://packagist.org/packages/masih/youtubedownloader#2.9.4
youtube video id (v= WkBNKa2KXZE,v=riorjrvLtg8)  is not downloading. and showing error "Masih \ YoutubeDownloader \ YoutubeException (20)
An error occurred when downloading video". How to resolve this.

Code:
```
$youtube = new YoutubeDownloader('WkBNKa2KXZE');
				$info = $youtube->getInfo();	
				$video_filename=$info->full_formats[0]->filename;
				$this->sanitizeFileName = function ($fileName) use ($youtube) {
					return str_replace('_', ' ', $youtube->pathSafeFilename($fileName));
				};				
				$save_path=base_path().'/assets/uploads/videos';
				$youtube->setPath($save_path);
				$youtube->onProgress = function ($downloadedBytes, $fileSize, $index, $count) {
					if ($count > 1) echo '[' . $index . ' of ' . $count . ' videos] ';
					if ($fileSize > 0)
						echo "\r" . 'Downloaded ' . $downloadedBytes . ' of ' . $fileSize . ' bytes [%' . number_format($downloadedBytes * 100 / $fileSize, 2) . '].';
					else
						echo "\r" . 'Downloading...'; // File size is unknown, so just keep downloading
				};
				$youtube->onComplete = function ($filePath, $fileSize, $index, $count) {
					if ($count > 1) echo '[' . $index . ' of ' . $count . ' videos] ';
					echo 'Downloading of ' . $fileSize . ' bytes has been completed. It is saved in ' . $filePath . PHP_EOL;
				};		
				$youtube->onFinalized = function ($filePath, $fileSize, $index, $count) {
					if ($count > 1) echo '[' . $index . ' of ' . $count . ' videos] ';
					echo $filePath . ' Finalized' . PHP_EOL;
				};										
				$youtube->download();
```

**Comment by masihyeganeh** [comment]
It is fixed in v2.9.5.
Please test and reopen this issue if the problem persists.
Thanks

**Comment by drpshtiwan** [comment]
The same problem for some videos exist.. Please fix it
